components/dashboard-migration-tool/src/main/java/org/wso2/carbon/dashboards/migrationtool/DSPortalAppMigrationTool.java

```diff
@@ -0,0 +1,258 @@
+/**
+ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * <p>
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.dashboards.migrationtool;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.log4j.BasicConfigurator;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+import java.io.*;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.security.cert.X509Certificate;
+
+public class DSPortalAppMigrationTool extends DSMigrationTool {
+    private static final Log log = LogFactory.getLog(DSPortalAppMigrationTool.class);
+    private static final String GADGET = "gadget";
+    private static final String WIDGET = "widget";
+    private static final String LAYOUT = "layout";
+    private static final String BLOCKS = "blocks";
+    private static final String INDEX_JSON = "index.json";
+    private static String storePath = "/repository/deployment/server/jaggeryapps/portal/store";
```

Comment:
It should be constant. Use final keyword as well. And the name should be all uppercase.